db/rdbms-schema/src/main/resources/db/changelog/rdbms-exporter/changesets/8.8.0.xml

```diff
@@ -619,7 +619,9 @@
         <constraints primaryKey="true"/>
       </column>
       <column name="EVENT_TIME" type="TIMESTAMP WITH TIME ZONE(3)"/>
-      <column name="ASSIGNEE_HASH" type="BIGINT"/>
+      <column name="ASSIGNEE_HASH" type="BIGINT">
+        <constraints primaryKey="true"/>
+      </column>
```

Comment:
👍 Nice. I just realized we don't have RDBMS exporter tests, there we would discover these earlier. I'll add some later.